v4.6.1-releasenotes.txt

```diff
@@ -0,0 +1,110 @@
+BISDN Linux
+v4.6.1 Release Notes
+Date: 2022-06-08
+
+! New in this release
+- baseboxd: updated to version 1.11.5
+- Linux kernel updated to version 5.15.44
+- stfp and sftp-server were added in preparation of OpenSSH 9.0+ upgrades on host systems (which changes the default protocol of scp to SFTP)
+
+! Bug fixes
+- baseboxd: a fix was added to properly handle default routes on data ports
+- meta-ofdpa: fixed the SFP LEDs on the Edgecore Networks EPS202 (AS4630-54PE)
```

Comment:
This has also a meta-open-network-linux part, though in the "high level" description it does not really matter where it was fixed, just what was fixed.

```suggestion
- Edgecore EPS202 (AS4630-54PE): fixed the SFP LEDs
```